### DISCUSSION TITLE
Can't move windows to other workspaces.

### DISCUSSION THREAD

**Discussion by LinArcX** [WayfireWM/wayfire#1549] [Q&A]
In https://github.com/WayfireWM/wayfire/wiki/Tutorial:

> Moving windows to workspaces
> To move a window to another workspace, simply press Super + Shift + [HJKL](https://en.wikipedia.org/wiki/Arrow_keys#HJKL_keys).

But i can't move any window to other workspaces.

**Comment by Cruising0904** [comment]
![image](https://user-images.githubusercontent.com/29267295/219368206-c8fbf26c-a001-4e94-8f5a-b3a1cb2765f6.png)
it works for me. I am not sure why yours is not working though. I had been through this before.. try 'binding_win_right' if it works
